i18n/fi.po

```diff
@@ -0,0 +1,1222 @@
+
+#: templates/features.html:82
+msgid "Integration Documentation"
+msgstr "Integrointidokumentit"
+
+#: templates/features.html:85
+msgid "Limits:"
+msgstr "Raja-arvot"
+
+#: templates/features.html:89
+msgid "Differentiation"
+msgstr "Derivointi:"
+
+#: templates/features.html:92
+msgid "Integration:"
+msgstr "Integrointi:"
+
+#: templates/features.html:93
+msgid "It uses extended Risch-Norman heuristic"
+msgstr "Se käyttää laajennettua Risch-Norman - heuristiikkaa"
+
+#: templates/features.html:95
+msgid "Taylor (Laurent) series"
+msgstr "Taylor (Laurent) - sarjat"
+
+#: templates/features.html:101
+msgid "Solving equations"
+msgstr "Yhtälönratkaisu"
+
+#: templates/features.html:105
+msgid "Polynomial equations"
+msgstr "Polynomiyhtälöt"
+
+#: templates/features.html:106
+msgid "Algebraic equations"
+msgstr "Algebralliset yhtälöt"
+
+#: templates/features.html:107
+msgid "Differential equations"
+msgstr "Differentiaaliyhtälöt"
+
+#: templates/features.html:108
+msgid "Difference equations"
+msgstr "Differenssiyhtälöt"
+
+#: templates/features.html:109
+msgid "Systems of equations"
+msgstr "Yhtälöryhmät"
+
+#: templates/features.html:110
+msgid "Diophantine equations:"
+msgstr "Diofantoksen yhtälöt"
+
+#: templates/features.html:116
+msgid "Combinatorics"
+msgstr "Kombinatoriikka"
+
+#: templates/features.html:120
+msgid "Permutations"
+msgstr "Permutaatiot"
+
+#: templates/features.html:121
+msgid "Combinations"
+msgstr "Kombinaatiot"
+
+#: templates/features.html:122
+msgid "Partitions"
+msgstr "Ositukset"
+
+#: templates/features.html:123
+msgid "Subsets"
+msgstr "Aliryhmät"
+
+#: templates/features.html:124
+msgid "Permutation Groups"
+msgstr "Permutaatioryhmät"
+
+#: templates/features.html:125
+msgid "Prufer and Gray Codes"
+msgstr "Prüfer ja Gray - koodit"
+
+#: templates/features.html:131
+msgid "Discrete math"
+msgstr "Diskreetti matematiikka"
+
+#: templates/features.html:134
+msgid "Binomial coefficients"
+msgstr "Binomikertoimet"
+
+#: templates/features.html:135
+msgid "Summations"
+msgstr "Summat"
+
+#: templates/features.html:136
+msgid "Products"
+msgstr "Tulot"
+
+#: templates/features.html:137
+msgid "Number theory:"
+msgstr "Lukuteoria:"
+
+#: templates/features.html:137
+msgid ""
+"generating prime numbers, primality testing,  integer factorization, "
+"Diophantine equation solving... "
+msgstr ""
+"alkulukujen generointi, alkulukutestit, kokonaislukuhajotelmat, diofantoksen "
+"yhtälöt..."
+
+#: templates/features.html:139
+msgid "Logic expressions"
+msgstr "Loogiset lauseet"
+
+#: templates/features.html:145
+msgid "Matrices"
+msgstr "Matriisit"
+
+#: templates/features.html:149
+msgid "Basic arithmetic"
+msgstr "Perusaritmetiikka"
+
+#: templates/features.html:150
+msgid "Eigenvalues/eigenvectors"
+msgstr "Ominaisarvot ja ominaisvektorit"
+
+#: templates/features.html:151
+msgid "Determinants"
+msgstr "Determinantit"
+
+#: templates/features.html:152
+msgid "Inversion"
+msgstr "Inversiot"
```

Comment:
I think  a literal translation would be "Kääntäminen", but that looks too general. Hence I suggest "Matriisien kääntäminen" or possibly "Käänteismatriisit".